tests/cancun/eip6780_selfdestruct/test_reentrancy_selfdestruct_revert.py

```diff
@@ -21,26 +24,123 @@
 REFERENCE_SPEC_VERSION = "2f8299df31bb8173618901a03a8366a3183479b0"
 
 
+def construct_call_op(call_type: Op, money: int, address_to: Address):
+    """Return call op based on call_type and address_to."""
+    if call_type in [Op.CALLCODE, Op.CALL]:
+        return call_type(Op.GAS, address_to, money, 0, 0, 0, 0)
+    else:
+        return call_type(Op.GAS, address_to, money, 0, 0, 0)
+
+
+@pytest.fixture
+def selfdestruct_contract_bytecode(selfdestruct_recipient_address: Address) -> Bytecode:
+    """Contract code that performs a SELFDESTRUCT operation."""
+    return Op.SELFDESTRUCT(selfdestruct_recipient_address)
+
+
+@pytest.fixture
+def selfdestruct_contract_init_balance() -> int:  # noqa: D103
+    return 300_000
+
+
+@pytest.fixture
+def selfdestruct_contract_address(
+    pre: Alloc, selfdestruct_contract_bytecode: Bytecode, selfdestruct_contract_init_balance: int
+) -> Address:
+    """Address of the selfdestruct contract."""
+    return pre.deploy_contract(
+        code=selfdestruct_contract_bytecode, balance=selfdestruct_contract_init_balance
+    )
+
+
+@pytest.fixture
+def executor_contract_bytecode(
+    first_suicide: Op,
+    revert_contract_address: Address,
+    selfdestruct_contract_address: Address,
+) -> Bytecode:
+    """Contract code that performs a selfdestruct call then revert."""
+    return (
+        Op.SSTORE(1, construct_call_op(first_suicide, 0, selfdestruct_contract_address))
+        + Op.SSTORE(2, Op.CALL(Op.GAS, revert_contract_address, 0, 0, 0, 0, 0))
```

Comment:
```suggestion
        + Op.SSTORE(2, Op.CALL(address=revert_contract_address))
```
We can also take the opportunity to simplify this one too.

Note that the `gas` parameter can be omitted only because `Op.GAS` is the default when the keyword argument is not present.